### ISSUE TITLE
Using multipart to upload a photo with an @reply does not work.

### ISSUE THREAD

**Issue by tommoor** [themattharris/tmhOAuth#46]
For example, the following code is unsuccessful: 

```
$response = $tmhOAuth->request(
      'POST',
      'https://upload.twitter.com/1/statuses/update_with_media.json',
      array(
          'media[]'  => '@' . $file,
          'status'   => '@example this is a great picture!',
      ),
      true, // use auth
      true  // multipart
```

   );

Obviously this is due to PHP's interpretation of the @ symbol at the beginning of the status field signifying a filepath. It would be great to get support for this within the library.

**Comment by tommoor** [comment]
I've just found that prepending a space to infront of the @ symbol in the status field allows the status to be sent. It seems that twitter does trim the space on their end as well although this feels like a frail solution.

**Comment by themattharris** [comment]
thanks for raising this, i'll see if I can work out what's going on here.

**Comment by themattharris** [comment]
interestingly this appears to be a fault in the way PHP works with libCurl. i'm looking for ways around this but there doesn't seem to be a 'nice' way to do it. i'll keep trying things though.

**Comment by themattharris** [comment]
shipped a way of handling this in 0.8.3
